### ISSUE TITLE
VVV 4

### ISSUE THREAD

**Issue by tomjn** [Varying-Vagrant-Vagrants/VVV#1785]
The plan for VVV 4 is to build on top of VVV 3 now that we can update the base box easily. When the VVV4 box gets built, we provision PHP/MariaDB/etc, so that when a user does their first provision, only the sites and tools need to be setup.

This should take a lot of the time out of setting things up, as well as eliminating a source of support burden so we can focus on making things nicer.

Additionally, if PHP or DB provisioning ever breaks, only the build server breaks, everybody carries on using the last built box

This also means we can eliminate a number of shared folders and folders on the filesystem, e.g. the `provision` and `config` folders. Only `database`, `logs`, `www`, and `certificates` might be needed, and those could be consolidated further ( perhaps into a subfolder? )

**Comment by benlumia007** [comment]
does this mean, that some software would eventually get pre-install inside the box rather than getting provisioned, that would make things faster

**Comment by tomjn** [comment]
yes, everything except the sites and certificats would be ready out of the
box

**Comment by IronGhost63** [comment]
I will no longer have "it's provisioning" excuse soon.
